test/fixtures/readme-with-custom-title.md

```diff
@@ -0,0 +1,212 @@
+# dockops [![build status](https://secure.travis-ci.org/thlorenz/dockops.png)](http://travis-ci.org/thlorenz/dockops)
```

Comment:
Please condense this fixture into a more minimal example.
This way it's hard to see what you're testing since it's so huge.